client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -276,6 +271,27 @@ function MetadataService(api, $q, staticMetadata) {
                 });
             }
         },
+        removeSubjectTerm: function(term) {
+            var o = {},
+                self = this,
+                subjectCodesArray = self.subjectScope.item[self.subjectScope.field],
+                temp = _.without(subjectCodesArray, term);
+
+            if (temp.length === subjectCodesArray.length) {
+                temp = removeSubjectCode(subjectCodesArray, term);
```

Comment:
Looking at this now, we could probably even get rid of the `removeSubjectCode()` function altogether and just replace it with the essential `_.remove(...)` line?
This will further increase the readability of the code and make it shorter.